tasks/configure.yml

```diff
@@ -51,6 +51,27 @@
     mode: 0755
     setype: mysqld_db_t
 
+- name: Create parent directory for mysql_pid_file if it does not exist
+  file:
+    path: "{{ mysql_pid_file.split('/')[0:-1]|join('/') }}"
```

Comment:
I don't think you need to create this directory, usually it will be auto-created by init system.
And, for instance, if your pid-file is `/var/lib` you will change ownership for whole `/var/run`